code/game/objects/items/weapons/material/thrown.dm

```diff
@@ -18,7 +18,13 @@
 	..()
 	if(material.radioactivity>0 && istype(hit_atom,/mob/living))
 		var/mob/living/M = hit_atom
-		M.adjustToxLoss(rand(20,40))
+		M.adjustToxLoss(rand(25,50))
 
 /obj/item/weapon/material/star/ninja

```

Comment:
Can we add a small chance here that in case this is thrown by someone thats not a ninja it has a small chance to hit themselfs (and a large chance if they have brain damage).

Imho it would be a good idea to slightly disencourage the usage by someone that just picked it up and wants to throw it back.

Also, can you catch them ? - I think that should be impossible (at least for someone that is not a ninja)

Reply:
They are auto shot from the module. You can't catch them either and I believe they go away if they miss.